mitosheet/css/toolbar.css

```diff
@@ -181,4 +181,21 @@
   padding: 0px 5px;
   color: var(--mito-background);
   height: 18px;
+}
+
+
+/* Tab Bar Style */
+.mito-toolbar-tabbar-button {
```

Comment:
rather than overloading button, maybe we rename this to a `tabbar-tabname` (and below).